test/SlowTests/Authentication/AuthenticationLetsEncryptTests.cs

```diff
@@ -61,6 +63,42 @@ public async Task CanGetLetsEncryptCertificateAndRenewIt()
 
             UseNewLocalServer();
             await RenewCertificate(serverCert, firstServerCertThumbprint);
+            using (Server.ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            using (context.OpenReadTransaction())
+            {
+                var local = Server.ServerStore.Cluster.GetCertificateThumbprintsFromLocalState(context).ToList();
+                Assert.Equal(0, local.Count);
+
+                var cluster = Server.ServerStore.Cluster.GetCertificateThumbprintsFromCluster(context).ToList();
+                Assert.Equal(0, cluster.Count);
+            }
+        }
+
+        [RetryFact(delayBetweenRetriesMs: 1000)]
+        public async Task CanGetLetsEncryptCertificateAndRenewAfterFailure()
+        {
+            var acmeUrl = "https://acme-staging-v02.api.letsencrypt.org/directory";
```

Comment:
change to Theory and also use pebble (check other tests, how they are doing this)

Reply:
done